src/l10n/chinese-simplified.ini

```diff
@@ -2,39 +2,59 @@
 ; The context of these messages is available here: https://github.com/oTranscribe/oTranscribe/wiki/Help-translate-oTranscribe#message-contexts
 help                = 帮助
 ; === homepage ===
-lead                = 轻松录制访问的免费网络应用程序
+lead                = 一个免费的工具，帮您摆脱听写录音稿的痛苦！
 start-loading       = 加载中...
-start-ready         = 开始录制
-mobile-warning      = oTranscribe 只能在桌上型电脑上使用。
-old-browser-warning = 您的浏览器不支持oTranscribe需要使用的HTML5功能。请换用一个新颖的浏览器。
-promo-1.innerHTML   = 播放器和编辑之间<strong>不需切换</strong>
-promo-2.innerHTML   = 双手不离键盘就能<strong>暂停，倒带，快进</strong>
-promo-3.innerHTML   =用<strong>互动时间戳</strong>导航您的抄本
-promo-4.innerHTML   = <strong>自动保存</strong>每秒自动保存到本地浏览器
-promo-5.innerHTML   = <strong>隐私保护</strong> - 您的音频文件和抄本不会离开您的电脑
-promo-6.innerHTML   = <strong>输出</strong>到Markdown,纯文本,和 Google Docs
-promo-7.innerHTML   = <strong>支持视频文件</strong>的集成播放器
-promo-8.innerHTML   = <strong><a href="opensource">开放源码</a></strong> 基于 MIT 许可证。
-press               = 刊登于以下媒体
+start-ready         = 开始听写输入
+mobile-warning      = oTranscribe 只能在桌面电脑上使用。
+old-browser-warning = 您的浏览器不支持 oTranscribe 所需的 HTML5 功能，请您更新为现代浏览器。
+promo-1.innerHTML   = <strong>无需来回切换</strong>您的播放器和编辑器。
+promo-2.innerHTML   = <strong>暂停、倒带、快进</strong>均使用键盘操作即可。
+promo-3.innerHTML   = <strong>时间标记交互</strong>，帮助您精确定位到稿件字句。
+promo-4.innerHTML   = <strong>自动保存</strong>，稿件每秒自动保存到本地浏览器。
```

Comment:
promo-4.innerHTML should probably be 保存到**你的**浏览器 (save to your browser) instead of 保存到本地浏览器 since 本地 generally means local as in geographically local rather than local on the network (i.e. your computer)